pkg/virt-controller/services/template.go

```diff
@@ -1355,7 +1354,12 @@ func checkForKeepLauncherAfterFailure(vmi *v1.VirtualMachineInstance) bool {
 	return keepLauncherAfterFailure
 }
 
-func requiresCPULimits(virtClient kubecli.KubevirtClient, labelSelector *metav1.LabelSelector, vmi *v1.VirtualMachineInstance) bool {
+func (t *templateService) doesVMIRequireAutoCPULimits(vmi *v1.VirtualMachineInstance) bool {
+	if namespaceWithRQ := t.doesVMIRequireAutoResourceLimits(vmi, k8sv1.ResourceCPU); namespaceWithRQ {
```

Comment:
```suggestion
	if t.doesVMIRequireAutoResourceLimits(vmi, k8sv1.ResourceCPU) {
```
Unless I'm missing something?

Reply:
Thank you!!! You are definitely right